src/v2/guide/migration.md

```diff
@@ -1540,47 +1540,47 @@ new Vue({
 
 ## Global Config
 
-### `Vue.config.debug` <sup>removed</sup>
+### `Vue.config.debug` <sup>removido</sup>
 
-No longer necessary, since warnings come with stack traces by default now.
+Ya no es necesario, ya que ahora las advertencias vienen con seguimiento de pila por defecto.
 
 {% raw %}
 <div class="upgrade-path">
-  <h4>Upgrade Path</h4>
-  <p>Run the <a href="https://github.com/vuejs/vue-migration-helper">migration helper</a> on your codebase to find examples of <code>Vue.config.debug</code>.</p>
+  <h4>Ruta de actualización</h4>
+  <p>Ejecute el <a href="https://github.com/vuejs/vue-migration-helper">asistente de migración</a> sobre su código para encontrar ejemplos de <code>Vue.config.debug</code>.</p>
 </div>
 {% endraw %}
 
-### `Vue.config.async` <sup>removed</sup>
+### `Vue.config.async` <sup>removido</sup>
 
-Async is now required for rendering performance.
+Ahora async es requerido para el rendimiento de la renderización.
 
 {% raw %}
 <div class="upgrade-path">
-  <h4>Upgrade Path</h4>
-  <p>Run the <a href="https://github.com/vuejs/vue-migration-helper">migration helper</a> on your codebase to find examples of <code>Vue.config.async</code>.</p>
+  <h4>Ruta de actualización</h4>
+  <p>Ejecute el <a href="https://github.com/vuejs/vue-migration-helper">asistente de migración</a> sobre su código para encontrar ejemplos de <code>Vue.config.async</code>.</p>
 </div>
 {% endraw %}
 
-### `Vue.config.delimiters` <sup>replaced</sup>
+### `Vue.config.delimiters` <sup>remplazado</sup>
```

Comment:
Typo:

```suggestion
### `Vue.config.delimiters` <sup>reemplazado</sup>
```

Actually, `remplazado` is accepted, but `reemplazado` is better for some reason 🤷‍♂